x-pack/test_utils/jest/contract_tests/__memorize_snapshots__/example.contract.test.ts.snap

```diff
@@ -103,3 +103,108 @@ exports['Example contract tests should run online or offline - example_test_snap
     "serverExists": true
   }
 }
+
```

Comment:
We don't plan on using jest contract test any more. Any chance you can blow away all of ` x-pack/test_utils/jest/contract_tests/` and all references? :) I think there's a few test setup places that are already in master if you grep for `jest_contract` and `contract_tests`

Reply:
What if we only removed any changes or references to `x-pack/test_utils/jest/contract_tests/` in this PR, but saved the "delete directory from master" for a separate PR?

Reply:
Works for me!